jcl/j9modules.xml

```diff
@@ -83,28 +89,69 @@
 					</else>
 				</if>
 			</case>
+			<case value="jdk.jconsole">
+				<replacerequires module.old="com.ibm.management" module.new="jdk.management" />
+			</case>
+			<case value="jdk.management">
+				<removeexportsoropens action="exports" package="com.sun.management.internal" />
+			</case>
+			<case value="openj9.sfj">
+				<replacerequires module.old="com.ibm.management" module.new="jdk.management" />
+			</case>
 			<default>
 				<echo message="No preparation required for module ${module.name}" />
 			</default>
 		</switch>
 	</target>
 
-	<macrodef name="removeexport">
+	<macrodef name="removeexportsoropens">
 		<attribute name="package" />
+		<attribute name="action" />
 		<sequential>
-			<echo message="Removing export of package @{package}" />
+			<echo message="Removing @{action} of package @{package}" />
 			<loadresource property="regexp.@{package}">
 				<string value="@{package}" />
 				<filterchain>
 					<replaceregex pattern="\." replace="\\\\." flags="g" />
 				</filterchain>
 			</loadresource>
 			<replaceregexp file="${module.dir}/classes/module-info.java" replace="" flags="gs">
```

Comment:
`${module.dir}/classes/module-info.java` might be defined as a variable and used across this xml file.